### ISSUE TITLE
create features scripts crash in the newest version

### ISSUE THREAD

**Issue by aokosinska** [KosinskiLab/AlphaPulldown#232]
When installing with pip:
```
python3 -m pip install alphapulldown==1.0.3
```
create_individual_features scripts crash with:
```
Traceback (most recent call last):
  File "/u/agobarsk/envs/AlphaPulldown/bin/create_individual_features_with_templates.py", line 7, in <module>
    from alphapulldown.utils import create_uniprot_runner, get_run_alphafold, convert_fasta_description_to_protein_name
  File "/u/agobarsk/envs/AlphaPulldown/lib/python3.10/site-packages/alphapulldown/utils.py", line 9, in <module>
    from alphafold.version import __version__ as AF_VERSION
ModuleNotFoundError: No module named 'alphafold.version'
```
The ```version.py``` is indeed missing from ```/u/agobarsk/envs/AlphaPulldown/lib/python3.10/site-packages/alphafold``` directory.

**Comment by DimaMolod** [comment]
thanks for pointing this out.
In the default AF version linked to AP the version.py file exists:
https://github.com/KosinskiLab/alphafold/blob/549bd360c53a857c8ab1b8e119092f7080971b6a/alphafold/version.py
Apparently, the submodules weren't updated before uploading the package to pypi. Need to fix ASAP!

**Comment by dingquanyu** [comment]
Now should be fixed in the new release.

**Comment by DimaMolod** [comment]
fixed in [5195998](https://github.com/KosinskiLab/AlphaPulldown/commit/519599841e98ec503957a2b4734980bd7db3de91). Closing...
